index.js

```diff
@@ -0,0 +1,19 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+async function run() {
+  try {
+    // `who-to-greet` input defined in action metadata file
+    const nameToGreet = core.getInput('who-to-greet');
```

Comment:
I think this inputs must match the definition in the YAML.

Reply:
I know, but this is not expected to succeed just yet. I wanted to get the boilerplate done before sending a pull request with the real code.